documentation/book/con-tls-connections.adoc

```diff
@@ -26,8 +26,8 @@ Like the Cluster Operator, the Topic and User Operators each use a TLS sidecar w
 
 == Kafka Client connections
 
-Encrypted communication between Kafka brokers and clients running within the same Kubernetes cluster is provided through the `CLIENTTLS` listener on port 9093.
+Encrypted communication between Kafka brokers and clients running within the same Kubernetes cluster is provided through the `tls` listener on port 9093.
```

Comment:
Something like "...can be provided by configuring the `spec.kafka.listeners.tls` listener, which listens on port 9093" is a bit more explicit about what exactly what `tls` we're referring to and that isn't not configured by default.